### PR TITLE
renamed blast easyconfigs (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by pescobar** [comment]
I reported this issue here https://github.com/hpcugent/easybuild-easyconfigs/issues/1442

**Comment by boegel** [comment]
@pescobar: two issues to fix:

```
EasyBuildError: 'EasyBuild crashed with an error (at easybuild/tools/robot.py:232 in
resolve_dependencies): Irresolvable dependencies encountered:
BLAST/2.2.28-goolf-1.4.10-Python-2.7.3, BLAST/2.2.28-ictce-5.3.0-Python-2.7.3'
```

```
AssertionError: Patch file /data/jenkins/workspace/easybuild-easyconfigs-pr-builder/
easybuild/easyconfigs/b/BLAST/BLAST+-2.2.30_ictce-fixes.patch is
available for BLAST+-2.2.30-goolf-1.4.10.eb
```

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 14 (13 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/efe4043216af0847dcd8 for a full test report.

File: easybuild/easyconfigs/b/BLAST/BLAST+-2.2.30-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'BLAST+'
+version = '2.2.30'
+altversions = ['2.2.26', '2.2.27'] # Versions .18 & .23-25 are also important but seem not to build fine, yet.
```
Comment (boegel) [inline easybuild/easyconfigs/b/BLAST/BLAST+-2.2.30-goolf-1.4.10.eb]: please drop this

File: easybuild/easyconfigs/b/BLAST/BLAST+-2.2.30-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+homepage = 'http://blast.ncbi.nlm.nih.gov/'
+description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
+ for comparing primary biological sequence information, such as the amino-acid
+ sequences of different proteins or the nucleotides of DNA sequences."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+# eg. ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.2.27/ncbi-blast-2.2.27+-src.tar.gz
```
Comment (boegel) [inline easybuild/easyconfigs/b/BLAST/BLAST+-2.2.30-goolf-1.4.10.eb]: drop outdated comment

**Comment by boegel** [comment]
@pescobar: please move the BLAST+ easyconfigs into a directory named BLAST+?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 11 out of 14 (13 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d2c6bc5c846990c72ea1 for a full test report.

**Comment by boegel** [comment]
trouble in paradise... will check later what went wrong

@pescobar: can you spot the problem, and maybe fix it?

**Comment by boegel** [comment]
@pescobar: `BLAST+-2.2.30-goolf-1.4.10.eb` doesn't work on my end...

```
/bin/sh: line 2: /usr/bin/basename: No such file or directory
/bin/ln: accessing `atic.a': No such file or directory
```

You're also triggering a rebuild of `HH-suite-2.0.16-goolf-1.4.10.eb`, which is known to be broken (see https://github.com/hpcugent/easybuild-easyconfigs/issues/414), but that's not the fault of this PR.

So, my suggestion would be: take out the new easyconfig for BLAST+ v2.2.30, and contribute that via a separate PR, so we can get the naming fix included for BLAST+.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 11 out of 13 (12 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f2e33e9d8687c844ce2c for a full test report.

**Comment by boegel** [comment]
All BLAST easyconfigs (still) work, the HH-suite failure is a known issue, so good to go.

Thanks @pescobar!

**Pull request merged**